src/main/java/io/github/moulberry/notenoughupdates/miscfeatures/WardrobeMouseButtons.kt

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (C) 2022 NotEnoughUpdates contributors
+ *
+ * This file is part of NotEnoughUpdates.
+ *
+ * NotEnoughUpdates is free software: you can redistribute it
+ * and/or modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation, either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * NotEnoughUpdates is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with NotEnoughUpdates. If not, see <https://www.gnu.org/licenses/>.
+ */
+package io.github.moulberry.notenoughupdates.miscfeatures
+
+import io.github.moulberry.notenoughupdates.NotEnoughUpdates
+import io.github.moulberry.notenoughupdates.core.config.KeybindHelper
+import io.github.moulberry.notenoughupdates.util.Utils
+import net.minecraft.client.gui.inventory.GuiChest
+import net.minecraftforge.client.event.GuiOpenEvent
+import net.minecraftforge.client.event.GuiScreenEvent
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
+
+class WardrobeMouseButtons {
+
+    private var keybinds: List<Int> = listOf(
+        NotEnoughUpdates.INSTANCE.config.wardrobeKeybinds.wardrobeSlot1,
+        NotEnoughUpdates.INSTANCE.config.wardrobeKeybinds.wardrobeSlot2,
+        NotEnoughUpdates.INSTANCE.config.wardrobeKeybinds.wardrobeSlot3,
+        NotEnoughUpdates.INSTANCE.config.wardrobeKeybinds.wardrobeSlot4,
+        NotEnoughUpdates.INSTANCE.config.wardrobeKeybinds.wardrobeSlot5,
+        NotEnoughUpdates.INSTANCE.config.wardrobeKeybinds.wardrobeSlot6,
+        NotEnoughUpdates.INSTANCE.config.wardrobeKeybinds.wardrobeSlot7,
+        NotEnoughUpdates.INSTANCE.config.wardrobeKeybinds.wardrobeSlot8,
+        NotEnoughUpdates.INSTANCE.config.wardrobeKeybinds.wardrobeSlot9,
+    )
+    private var lastClick = -1L
+
+    @SubscribeEvent
+    fun onGui(event: GuiScreenEvent) {
+        if (!NotEnoughUpdates.INSTANCE.config.wardrobeKeybinds.enableWardrobeKeybinds) return
+        val gui = event.gui as? GuiChest ?: return
+        if (!Utils.getOpenChestName().contains("Wardrobe")) return
+
+        for (i in keybinds.indices) {
+            if (KeybindHelper.isKeyDown(keybinds[i])) {
+                if (System.currentTimeMillis() - lastClick > 300) {
+                    Utils.sendLeftMouseClick(gui.inventorySlots.windowId, 36 + i)
+                }
+                lastClick = System.currentTimeMillis()
```

Comment:
Move this line one line up / only save the last click when we actually send a click.

When switching to different armour sets quickly, this behaviour feels wrong.